client-react/src/features/todos/models/todoAction.model.ts

```diff
@@ -1,4 +1,6 @@
+
```

Comment:
Keep naming convension the same accross the project. Use kebab-case

Reply:
fixed